portfolio/src/main/webapp/index.html

```diff
@@ -60,8 +60,25 @@ <h1>Kwame Ampadu-Boateng's Portfolio</h1>
       </p>
       <button onclick="addRandomGreeting()">Explore</button>
       <div id="greeting-container"></div>
-      <button onclick="getData()">Fetch a quote!</button>
+
+      <br />
```

Comment:
I think <br> is sufficient if you only want a newline (line-break).

Reply:
Sorry, I am confused by this comment.

Reply:
Ooops I typed ```<br>``` and obviously it was automatically interpreted as a line-break.

Reply:
I just checked this out. This is also a strange thing prettier does, it changes all of my `<br>` to `<br />`